postgres_insert_block.py

```diff
@@ -0,0 +1,62 @@
+from nio.properties import BoolProperty, StringProperty
+from nio.util.discovery import discoverable
+from .postgres_base_block import PostgresBase
+
+
+@discoverable
+class PostgresInsert(PostgresBase):
+    """A block for inserting incoming signals into a postgres database.
+
+    Properties:
+        table_name(str): name of the table on the database to execute commands
+                         on.
+
+    """
+
+    upsert = BoolProperty(title="Update conflicting keys", default=True)
+    table_name = StringProperty(title="Table name", allow_none=False)
```

Comment:
This probably can go in the base block, pretty much every connection will probably talk to one and only one table